CheatSheets/git-cheatsheet.md

```diff
@@ -66,6 +66,8 @@ created: 2022-10-18
 | `gitk` | Shows graphical interface for a local repository |
 | `git revert [commit-ID]` | Create new commit, reverting the changes from a specified commit |
 | `git checkout [file]` | Matches the file with last commit |
+| `git help` | Shows list of the most commonly used Git commands |
+| `git restore` | Restore working tree files |
```

Comment:
```suggestion
| `git restore .` | - Restore working tree files <br /> - To restore all files in the current directory |
```
| Command | Description |
| ------------ | ------------ |
| `git restore .` | - To restore all files in the current directory <br /> - Restore working tree files|